src/components/Quotes.jsx

```diff
@@ -0,0 +1,61 @@
+import { useEffect, useState } from 'react';
+
+const Quotes = () => {
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState();
+  const [data, setData] = useState([]);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      setLoading(true);
+      const url = 'https://api.api-ninjas.com/v1/quotes';
+      const options = {
+        method: 'GET',
+        headers: {
+          'content-type': 'application/json',
+          'X-Api-Key': 'eYPIRhhkNTX2LPyC2m6a6A==psZgcCiykqUJROEl',
+        },
+      };
+
+      try {
+        const response = await fetch(url, options);
+        const result = await response.json();
+        setData(result);
+        setLoading(false);
+      } catch (error) {
+        setError(error);
+      }
+    };
+    fetchData();
+  }, []);
+
+  return (
+    <>
+      <ul>
+        {loading ? (
+          <>Loading . . .</>
+        ) : (
+          <>
+            {data.map((item) => (!error ? (
+              <>
+                <div key={null}>
+                  <li>{item.quote}</li>
+                  <br />
+                  <li>
+                    ~
+                    {item.author}
+                  </li>
+                  {error}
+                </div>
+              </>
```

Comment:
- [ ] Each child in a list should have a valid `key` prop as you can see in the console error below
![image](https://user-images.githubusercontent.com/46839548/235265958-573926a9-7c47-4ca1-9eb6-641644e2d38b.png)

My suggestion is that you don't need to map through the data since it's always an array that contains only 1 item. You can just say `data[0].quote` and `data[0].author` instead 😉 
If you insist you want to map, then use `uuid` to ensure that you give unique valid key props to every child in the list 👍